tests/index.html

```diff
@@ -10,11 +10,21 @@
   <div id="qunit"></div>
   <div id="qunit-fixture"></div>
 
+  <script src="/sdk-testing.js"></script>
+  <!-- this needs thought but it brings in polyfills for the amd build :( -->
+  <script src="shopify-buy.umd.polyfilled.js"></script>
+  <script src="/qunit-shim.js"></script>
+  <script src="/shopify-buy.amd.js"></script>
+  <script src="/eslint-tests.js"></script>
+  <script src="/tests.js"></script>
```

Comment:
Always hate test runners where you need to load up specific script tags in a test HTML page, feels so weird in the era of JS modules.